WordPress/Classes/Extensions/Array.swift

```diff
@@ -20,6 +20,24 @@ extension Array where Element: Hashable {
     }
 }
 
+
+extension Array {
+    /// Returns array containing only elements that have unique values for item at given KeyPath.
+    /// Useful when you want to `unique` objects by their property, e.g. find all unique `author`s for a list of `Comment`s.
+    ///
+    /// - Parameter filteringKeyPath: A KeyPath to a `Hashable` property that the elements should be uniqued by.
+    ///
+    /// - Returns: Array with values filtered by using the given KeyPath to ensure only unique values are present.
+    public func unique<Value: Hashable>(by filteringKeyPath: KeyPath<Element, Value>) -> [Element] {
```

Comment:
Documentation is good, but is this still being used?

Reply:
It's not, but you said previously that you'd be ok with keeping it if it has docs ;)

Reply:
I see, sorry for the confusion. I mean I was ok with it if it was to be used. It’s a good extension that might be useful in the future, but I don’t think we want to keep unused code around

Reply:
@koke sure, no worries. gotten rid of it now :)